.changeset/pink-pillows-dream.md

```diff
@@ -0,0 +1,5 @@
+---
+'@wagmi/core': patch
+---
+
+Added getProvider to InjectedConnector options
```

Comment:
Nice changeset! Maybe we can just add some quick usage as well.

```suggestion
Added `getProvider` to `InjectedConnector` options.

Example: 

```ts
import { InjectedConnector } from '@wagmi/core/connectors/injected'

const connector = new InjectedConnector({
  options: {
    name: 'SubWallet',
    getProvider: () =>
      typeof window !== 'undefined' ? window.SubWallet : undefined,
  },
})
```